css/header.css

```diff
@@ -0,0 +1,273 @@
+.nav {
+  background: #fff;
+  position: fixed;
+  top: 0;
+  padding: 1rem;
+  overflow: hidden;
+  margin-bottom: 1rem;
+  width: 100%;
+  box-shadow: 0 1px 4px #2e2828;
+  z-index: 10;
+}
+
+.nav .menu {
+  color: var(--primary-color);
+  width: fit-content;
+  background: none;
+  border: none;
+}
+
+.nav .nav-items {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background-color: var(--secondary-color);
+  flex-direction: column;
```

Comment:
- About and Home: Mobile size: 
- The pages should look almost identical to the original design: colors, text, backgrounds, etc ...
- 
![Capture](https://user-images.githubusercontent.com/63972670/203527359-d78a0dce-91ba-4025-a616-ce95d08beb9e.PNG)
- **Yours, you need to change the top color. much the color to the main top**

![1](https://user-images.githubusercontent.com/63972670/203527857-ab110947-cd54-4a38-a314-6f2c1999e473.PNG)